build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
     id 'de.undercouch.download' version '4.0.0'
     id 'io.spring.dependency-management' version '1.0.10.RELEASE'
     id 'org.hidetake.swagger.generator' version '2.18.2'
-    id 'org.springframework.boot' version '2.3.1.RELEASE'
```

Comment:
2.3.1 had some issues with the Azure libs -- 2.4.0 works fine, and is what WSM uses.